### ISSUE TITLE
error: implicit declaration of function 'XXX' is invalid in C99 

### ISSUE THREAD

**Issue by dlavrov** [UCSC-LoweLab/tRNAscan-SE#6]
When I try to build tRNAscan-SE on my Mac (Catalina + Xcode 12.0) it fails with the following error:

`src/eufind_main.c:106:21: error: implicit declaration of function 'getopt' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
  while ((optchar = getopt(argc, argv, OPTIONS)) != -1)
                    ^
src/eufind_main.c:106:21: note: did you mean 'Getopt'?
src/sqfuncs.h:63:12: note: 'Getopt' declared here
extern int Getopt(int argc, char **argv, 
           ^
src/eufind_main.c:168:17: error: implicit declaration of function 'IntEncodeSeq' is invalid in C99
      [-Werror,-Wimplicit-function-declaration]
    if (errno = IntEncodeSeq(iseq,seq,sqinfo.len))
                ^
src/eufind_main.c:168:15: warning: using the result of an assignment as a condition without parentheses [-Wparentheses]
    if (errno = IntEncodeSeq(iseq,seq,sqinfo.len))
        ~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/eufind_main.c:168:15: note: place parentheses around the assignment to silence this warning
    if (errno = IntEncodeSeq(iseq,seq,sqinfo.len))
              ^
        (                                        )
src/eufind_main.c:168:15: note: use '==' to turn this assignment into an equality comparison
    if (errno = IntEncodeSeq(iseq,seq,sqinfo.len))
              ^
              ==
src/eufind_main.c:175:7: error: implicit declaration of function 'Init_tRNA' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
      Init_tRNA(prev_tRNA);     /* clear previous tRNA */
      ^
src/eufind_main.c:194:14: error: implicit declaration of function 'GetBbox' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
      while (GetBbox(&FirstScore,&seqidx,iseq,sqinfo.len,strand,
             ^
`
There appear to be similar problems across multiple packages with Xcode 12.0, e.g., https://github.com/astropy/astropy/issues/10770

**Comment by patriciaplchan** [comment]
Thanks for sending the error messages. Xcode 12.0 changes to raise the missing function declaration as an error instead of warning. Unfortunately, this is rather common in old C programs. We are going to include a fix in the next maintenance update that will be released in the next couple of weeks. You can wait for it or possibly just comment out the eufindtRNA in the Makefile, as it is not used for the default, new v2.0 algorithm (it is just included to run the legacy version of v1.3 if needed by users). We will tweet at https://twitter.com/UCSCtRNA when the new release is available.

**Comment by patriciaplchan** [comment]
Fix is included in v2.0.8.
